addons/website/static/tests/tours/parallax.js

```diff
@@ -14,15 +14,11 @@ wTourUtils.registerWebsitePreviewTour("test_parallax", {
     wTourUtils.changeOption("BackgroundOptimize", "we-toggler"),
     wTourUtils.changeOption("BackgroundOptimize", 'we-button[data-gl-filter="blur"]'),
 {
-    content: "Check that the Cover snippet has the Blur filter on its background image",
-    trigger: ":iframe .s_cover span[data-gl-filter='blur']",
+    content: "Check that the background image of the Cover snippet has been modified",
+    trigger: ":iframe .s_cover span.o_modified_image_to_save",
```

Comment:
As said in another comment: make sure to do better than this. On the python side of the test, you can check the actual record that were created/updated to save those applied options.